timer.c

```diff
@@ -55,7 +57,7 @@ lk_time_t current_time(void)
 
     rdtsc(low, high);
     val = (uint64_t)((uint64_t)high <<32 | (uint64_t)low);
-    timer_current_time = val/(uint64_t)(g_trusty_startup_info.calibrate_tsc_per_ms);
+    timer_current_time = val / TSC_PER_MS;
```

Comment:
why hard-coded?

Reply:
startup info have been removed, so there is no way to get tsc calibration unless use PIT, so  use it as hardcore for now.

We have self tsc calibration in latest Google Trusty reference code base.